### PR TITLE
ビルド日時が正しく取れていなかったのを修正した

### PR DESCRIPTION
**Pull request by tumugin**

### CODE REVIEW

File: NowPlayingV2/BuildDateAttribute.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
```
Comment (myskng) [inline NowPlayingV2/BuildDateAttribute.cs]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tumugin/NowPlayingV2/blob/cf5483954ef672cfd25a86ac2a647cc903ed4433/NowPlayingV2/BuildDateAttribute.cs#L2"></span>Using directive is not required by the code and can be safely removed</td>
    </tr>
  </tbody>
</table>

File: NowPlayingV2/BuildDateAttribute.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+using System.Globalization;
+using System.Text;
```
Comment (myskng) [inline NowPlayingV2/BuildDateAttribute.cs]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tumugin/NowPlayingV2/blob/cf5483954ef672cfd25a86ac2a647cc903ed4433/NowPlayingV2/BuildDateAttribute.cs#L4"></span>Using directive is not required by the code and can be safely removed</td>
    </tr>
  </tbody>
</table>

File: NowPlayingV2/BuildDateAttribute.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+[AttributeUsage(AttributeTargets.Assembly)]
+internal class BuildDateAttribute : Attribute
```
Comment (myskng) [inline NowPlayingV2/BuildDateAttribute.cs]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tumugin/NowPlayingV2/blob/cf5483954ef672cfd25a86ac2a647cc903ed4433/NowPlayingV2/BuildDateAttribute.cs#L7"></span>Namespace does not correspond to file location, should be: ‘NowPlayingV2’</td>
    </tr>
  </tbody>
</table>

File: NowPlayingV2/BuildDateAttribute.cs [hunk 4]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Globalization;
+
+[AttributeUsage(AttributeTargets.Assembly)]
+internal class BuildDateAttribute : Attribute
```
Comment (myskng) [inline NowPlayingV2/BuildDateAttribute.cs]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tumugin/NowPlayingV2/blob/93b5d464cbe4708425716316093fe03065986fea/NowPlayingV2/BuildDateAttribute.cs#L5"></span>Namespace does not correspond to file location, should be: ‘NowPlayingV2’</td>
    </tr>
  </tbody>
</table>

Comment (tumugin) [reply NowPlayingV2/BuildDateAttribute.cs]: これはしょうがないね

Comment (tumugin) [reply NowPlayingV2/BuildDateAttribute.cs]: いや、しょうがなくなくないわ
直す

**Pull request merged**